src/main/java/taxi/service/RegistrationServiceImp.java

```diff
@@ -0,0 +1,32 @@
+package taxi.service;
+
+import taxi.dao.DriverDao;
+import taxi.exception.RegistrationException;
+import taxi.lib.Inject;
+import taxi.lib.Service;
+
+@Service
+public class RegistrationServiceImp implements RegistrationService {

```

Comment:
Just to understand more clearly.
What was this service designed for?

Reply:
For registration a new driver, This is my addition, which is not provided by the terms of the task. This service is to register a new driver, to check if there is a driver with the same login, and to check if the repeated password matchesc

Reply:
I think this is an important point in the registration process, which should not be considered a mistake